sycl/test-e2e/syclcompat/memory/memory_management_test3.cpp

```diff
@@ -482,6 +482,125 @@ template <typename T> void test_fill_async_q() {
   free(h_A);
 }
 
+void constant_memcpy_async() {
+  std::cout << __PRETTY_FUNCTION__ << std::endl;
```

Comment:
Since the `constant_memcpy` tests here are similar to those in `memory_management_test1.cpp`, the comments I left there also apply here.